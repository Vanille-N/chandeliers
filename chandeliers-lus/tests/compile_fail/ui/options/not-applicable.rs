//! Annotation on a construct that doesn't accept them.
//! (`#[main]` is only for `node`)
chandeliers_lus::decl! {
    #[main]
    const X : int = 0;
}

fn main() {}
