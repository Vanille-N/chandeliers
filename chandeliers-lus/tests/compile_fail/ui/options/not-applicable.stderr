error: The attribute #[main(nb_iter)] is not valid here (does not apply to Const)
 --> tests/fail/ui/options/not-applicable.rs:2:5
  |
2 |     #[main]
  |     ^^^^^^^
